engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceTest.java

```diff
@@ -487,4 +491,300 @@ public void shouldCompletedProcessInstanceWhenElementsInsideExistingFlowScopesAr
         .describedAs("Expect the process instance to have been completed")
         .isPresent();
   }
+
+  @Test
+  public void shouldTerminateElementInRootScope() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            Bpmn.createExecutableProcess(PROCESS_ID)
+                .startEvent()
+                .serviceTask("A", a -> a.zeebeJobType("A"))
+                .endEvent()
+                .done())
+        .deploy();
+
+    final var processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+    final var elementInstanceKey =
+        RecordingExporter.processInstanceRecords(ProcessInstanceIntent.ELEMENT_ACTIVATED)
+            .withProcessInstanceKey(processInstanceKey)
+            .withElementId("A")
+            .getFirst()
+            .getKey();
+
+    // when
+    ENGINE
+        .processInstance()
+        .withInstanceKey(processInstanceKey)
+        .modification()
+        .terminateElement(elementInstanceKey)
+        .modify();
+
+    // then
+    assertThatElementIsTerminated(processInstanceKey, "A");
+    assertThatJobIsCancelled(processInstanceKey, "A");
+  }
+
+  @Test
+  public void shouldTerminateMultipleElementsInRootScope() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            Bpmn.createExecutableProcess(PROCESS_ID)
+                .startEvent()
+                .parallelGateway()
+                .serviceTask("A", a -> a.zeebeJobType("A"))
+                .endEvent()
+                .moveToLastGateway()
+                .serviceTask("B", b -> b.zeebeJobType("B"))
+                .endEvent()
+                .done())
+        .deploy();
+
+    final var processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+    final var elementsInstanceKeys =
+        RecordingExporter.processInstanceRecords(ProcessInstanceIntent.ELEMENT_ACTIVATED)
+            .withProcessInstanceKey(processInstanceKey)
+            .withElementType(BpmnElementType.SERVICE_TASK)
+            .limit(2)
+            .collect(Collectors.toMap(r -> r.getValue().getElementId(), Record::getKey));
+
+    // when
+    ENGINE
+        .processInstance()
+        .withInstanceKey(processInstanceKey)
+        .modification()
+        .terminateElement(elementsInstanceKeys.get("A"))
+        .terminateElement(elementsInstanceKeys.get("B"))
+        .modify();
+
+    // then
+    assertThatElementIsTerminated(processInstanceKey, "A");
+    assertThatElementIsTerminated(processInstanceKey, "B");
+    assertThatJobIsCancelled(processInstanceKey, "A");
+    assertThatJobIsCancelled(processInstanceKey, "B");
+  }
+
+  @Test
+  public void shouldTerminateElementWithIncidentOnJob() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            Bpmn.createExecutableProcess(PROCESS_ID)
+                .startEvent()
+                .serviceTask("A", a -> a.zeebeJobType("A")) // invalid expression
```

Comment:
:wrench: We should fix the comment. 

There is no expression. As the test method points out, the incident is created because the job is failed.

```suggestion
                .serviceTask("A", a -> a.zeebeJobType("A")) 
```